guides/v2.1/config-guide/bootstrap/magento-bootstrap.md

```diff
@@ -17,30 +17,30 @@ functional_areas:
 
 <h2 id="config-boot-logic">Bootstrap run logic</h2>
 
 <a href="{{ site.mage2000url }}app/bootstrap.php" target="_blank">The bootstrap object</a> uses the following algorithm to run the Magento application:
 
-1.	Initializes the error handler.
-2.	Creates the <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/ObjectManager" target="_blank">object manager</a> and basic shared services that are used everywhere and are affected by the environment. The environment parameters are injected properly into these objects.
-3.	Asserts that maintenance mode is *not* enabled; otherwise, terminates.
-4.	Asserts that the Magento application is installed; otherwise, terminates.
-5.	Starts the Magento application.
+1.  Initializes the error handler.
+2.  Creates the <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/ObjectManager" target="_blank">object manager</a> and basic shared services that are used everywhere and are affected by the environment. The environment parameters are injected properly into these objects.
+3.  Asserts that maintenance mode is *not* enabled; otherwise, terminates.
+4.  Asserts that the Magento application is installed; otherwise, terminates.
+5.  Starts the Magento application.
 
-	Any uncaught exception during application launch is automatically passed back to Magento in the `catchException()` method which you can use to handle the exception. The latter must return either `true` or `false`:
+  Any uncaught exception during application launch is automatically passed back to Magento in the `catchException()` method which you can use to handle the exception. The latter must return either `true` or `false`:
 
-    *	If `true`: Magento handled exception successfully. No need to do anything else.
-    *	If `false` (or any other empty result): Magento did not handle the exception. The bootstrap object performs the default exception handling subroutine.
-6.	Sends the response provided by the application object.
+    * If `true`: Magento handled exception successfully. No need to do anything else.
+    * If `false` (or any other empty result): Magento did not handle the exception. The bootstrap object performs the default exception handling subroutine.
+6.  Sends the response provided by the application object.
 
 <div class="bs-callout bs-callout-info" id="info">
 <span class="glyphicon-class">
-  <p>The assertions that the Magento application is installed and not in maintenance mode is the default behavior of the <code>\Magento\Framework\App\Bootstrap</code> class. You can modify it using an entry point script when creating the bootstrap object.</p></span>
+  <p>The assertions that the Magento application is installed and not in maintenance mode is the default behavior of the <a href="{{ site.mage2100url }}lib/internal/Magento/Framework/App/Bootstrap.php" target="_blank">\Magento\Framework\App\Bootstrap</a> class. You can modify it using an entry point script when creating the bootstrap object.</p></span>
```

Comment:
This is a correct HTML link, because it's within an HTML info note. No change needed.